Assets/MixedRealityToolkit.Extensions/SpectatorView/Plugins/x64/.gitignore

```diff
@@ -0,0 +1 @@
+!*.dll.meta
```

Comment:
After thinking about this, we probably don't want these meta files here. I think we do want them checked in near the native binaries in the mixedrealitycompanionkit. But the meta files should be dropped into the project from the nuget package that those binaries could be placed in.

Reply:
OK, removed those!